recipe/meta.yaml

```diff
@@ -15,8 +17,8 @@ source:
       - fix_ppc64le.patch  # [ppc64le]
 
 build:
-  number: 1
-  string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+  number: 2
+  string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
```

Comment:
The whole idea of separating librealsense and pyrealsense2 packages is that librealsense does not depend on Python. However, to do so we should have a `ignore_run_exports_from:
    - python` block, see https://github.com/conda-forge/opencv-feedstock/blob/891639f983996e8f7cae4595b53d7293cd31058b/recipe/meta.yaml#L59C1-L60C13 for example.

Reply:
Missed that, added.